Cura/util/printerConnection/printerConnectionBase.py

```diff
@@ -51,6 +51,10 @@ def getName(self):
 		return self._name
 
 	#Load the data into memory for printing, returns True on success
```

Comment:
Should perhaps have an extra line of comment here, explaining how/why this method differs from `loadGCodeData`.